lib/services/storage_service.ts

```diff
@@ -360,10 +363,11 @@ export class SNStorageService extends PureService {
     });
   }
 
-  public async clearAllData() {
-    return Promise.all([
-      this.clearValues(),
-      this.clearAllPayloads()
-    ]);
+  public clearAllData(): Promise<void> {
+    return this.executeCriticalFunction(async () => {
+      await this.clearValues();
+      await this.clearAllPayloads();
+      await this.deviceInterface!.removeAllRawStorageValues();
```

Comment:
This might interfere with namespacing. When you sign out of one account, we don't want to sign out of all accounts.